CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Allows explicit specification of the plotter executable path if this is preferred over setting the `PATH` environment variable to find the program.
   Presently does not support executables other than the expected names (`chia`, and `chia_plot`).
   ([#823](https://github.com/ericaltendorf/plotman/pull/823))
+- Added new plot configuration options to madmax (--bucket3 & --rmulti2 )
```

Comment:
This also needs to be moved up into the new unreleased changelog section, but GitHub won't let me suggest up there...

```markdown
- `-v`/`--buckets3` and `-K`/`--rmulti2` are configurable for madMAx.
  ([#869](https://github.com/ericaltendorf/plotman/pull/869))
```

Reply:
Replied on Keybase :-)